.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: java
+sudo: required
+dist: xenial
```

Comment:
did you check if it works, Travis docs say xenial is experimental and dockerized builds are better if you need more modern ubuntu available during build

Reply:
I did, look at the build status - it's green :-)